website/views/manage_carousel.py

```diff
@@ -0,0 +1,120 @@
+from django.shortcuts import render, redirect
```

Comment:
I am trying to maintain PEP8 . So should the imports be like
`from django.shortcuts import redirect, render`

or is it with the other imports like
```
from website.forms import AddEditCarouselImageForm

from website.models import CarouselImage

from .tools import has_commit_permission
```